storm-server/src/main/java/org/apache/storm/scheduler/IsolationScheduler.java

```diff
@@ -218,8 +218,8 @@ public void schedule(Topologies topologies, Cluster cluster) {
 
         List<ExecutorDetails> allExecutors = new ArrayList<ExecutorDetails>();
         Collection<List<ExecutorDetails>> values = compExecutors.values();
-        for (List<ExecutorDetails> eList : values) {
-            allExecutors.addAll(eList);
+        for (List<ExecutorDetails> value : values) {
+            allExecutors.addAll(value);
```

Comment:
What was the violation here?

Reply:
`Local variable name 'eList' must match pattern '^[a-z]([a-z0-9][a-zA-Z0-9]*)?$'. [LocalVariableName]` which is good thing imo

Reply:
OK, and it did not start with the minimum 2 lower-case letters. Thanks!